src/backend/games.ts

```diff
@@ -1,3 +1,4 @@
+import { InstallPlatform } from './../common/types'
```

Comment:
Might want to import this after `ProtonVerb` below (l9/10)

Reply:
fixed